frontend/src/lib/components/Admin/EditQuestionModal.svelte

```diff
@@ -37,6 +37,18 @@ let {
 } = $props();
 let preview_lang = $state("de");
 let preview_answer = $state("");
+let options = $derived.by(() => {
+	let opts = [{ value: "", name: "No free text option" }];
+	if (!question || !question.options) {
+		return opts;
+	}
+	return opts.concat(
+		question.options.split(";").map((value) => ({
```

Comment:
```suggestion
		question.options.split(";").map((value) => ({
```

```suggestion
		question.options.replace(/;$/, '').split(";").map((value) => ({
```

In order to prevent a trailing ";" creating a fake option

![image](https://github.com/user-attachments/assets/f7520aa0-fa15-42b9-bb4f-469494df6f38)